CNAME

```diff
@@ -1 +0,0 @@
-rdiff-backup.org
```

Comment:
We're going to want to keep this file, so that GitHub pages can serve content at http://rdiff-backup.org directly.